src/shared/logs-show.c

```diff
@@ -485,6 +488,33 @@ static int output_short(
                 fprintf(f, ": [%s blob data]\n", format_bytes(bytes, sizeof(bytes), message_len));
         } else {
                 fputs(": ", f);
+
+                /* URLify config_file string in message, if the message starts with it.
+                 * Skip URLification if the highlighted pattern overlaps. */
+                if (config_file &&
+                    message_len >= config_file_len &&
+                    memcmp(message, config_file, config_file_len) == 0 &&
```

Comment:
i would feel more comfortable if we'd check that the next following char is either ":" or space, btw

Reply:
hmm, btw, we have memory_startswith() for cases like this

Reply:
memory_startswith() doesn't work here because both strings are unterminated.

I'll add a check for ':/ /\0'.